mint.json

```diff
@@ -146,7 +146,8 @@
         "content/components/frames",
         "content/components/steps",
         "content/components/tabs",
-        "content/components/tooltips"
+        "content/components/tooltips",
+        "content/components/mermaid-diagrams"
```

Comment:
Can we order it alphabetically?

Reply:
Done.

<img width="367" alt="Screenshot 2024-03-08 at 5 58 37 AM" src="https://github.com/mintlify/docs/assets/6391763/b781aba0-d656-4b6a-80c8-edb15b92bb45">